### PR TITLE
fix(chat): fix app log visibility for Admin, disable 'Use application' button (Issue #2526, #2531)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Need to disable Use application button for Code apps in state Undeployed
Application log for published and deployed Code app should be available for Admin only

Issues:

- Issue #2526 
- Issue #2531 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx [hunk 1]
```diff
@@ -224,17 +226,18 @@ export const ApplicationDetailsFooter = ({
...
-              <button
-                onClick={() => onLogsClick(entity)}
-                className="icon-button"
-                data-qa="application-logs"
-              >
-                <IconFileDescription size={24} />
-              </button>
-            </Tooltip>
-          )}
+          {((isPublicApp && isAdmin) || isMyApp) &&
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx]:
```suggestion
          {(isAdmin || isMyApp) &&
```

File: apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx [hunk 2]
```diff
@@ -144,23 +144,25 @@ export const ApplicationDetailsFooter = ({
...
-                    playerStatus === SimpleApplicationStatus.DEPLOY,
-                })}
-                data-qa="application-status-toggler"
-              >
-                <PlayerIcon size={24} />
-              </button>
-            </Tooltip>
-          )}
+          {((isPublicApp && isAdmin) || isMyApp) &&
+            isExecutable &&
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx]: `isExecutable` already checked `isMyApp || isAdmin`.
Please revert this block

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>- **Deployment failed**: [canceled](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/534842)

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2540.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**